server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -144,6 +144,8 @@ class ChallengeFlow extends BaseStep {
 
         defendersToKneel = undefined;
 
+        this.game.raiseEvent('onDefendersDeclared', { player: this.challenge.defendingPlayer, numOfDefendingCharacters: defenders.length });
```

Comment:
Is there a way we can avoid introducing this event? We already got rid of an event very similar to this.

Reply:
I´ve implemented this event for Left-Hand Lucas Codd. His reaction triggers to a player not declaring ANY defenders. As far as i have found out, there is an event for a character being declared as a defender (onDeclaredAsDefender) but those do not get created when there are no defenders declared.
Do you see another way to implement this kind of triggering conditon?

Reply:
The ability is sufficiently strange, it's either this or a dedicated "declared no defenders" event. It's probably fine for the moment, lets just try not to rely on it too much.